templates/signup-teacher.html

```diff
@@ -119,9 +121,9 @@ <h4 class="text-blue-500 mt-0 mb-2 text-center">{{_('create_account_explanation'
             </div>
             <!-- The extra check options -->
             <div class="border border-gray-400 rounded-lg mx-4 px-4 md:px-8 flex flex-col gap-1 py-2">
-                <div class="flex flex-row justify-between items-center">
+                <div class="flex flex-row justify-between items-center hidden">
                     <label for="is_teacher">{{_('request_teacher')}}</label>
-                    <input type="checkbox" id="is_teacher" name="is_teacher">
+                    <input type="checkbox" id="is_teacher" name="is_teacher" checked>
```

Comment:
Why not keep the one-page structure, check/uncheck the checkbox based on the flow selection of the user. It can always stay hidden, that's fine.